packages/x-components/src/components/suggestions/base-suggestion.vue

```diff
@@ -111,6 +119,17 @@
       return !!this.query && normalizeString(this.suggestion.query).includes(this.query);
     }
 
+    /**
+     * Checks if the suggestion is curated and if it should be highlighted.
+     *
+     * @returns True if the suggestion is curated and should be highlighted.
+     *
+     * @internal
+     */
+    protected get shouldHighlightCurated(): boolean {
+      return this.highlightCurated && (this.suggestion.isCurated ?? false);
```

Comment:
I know that this is copied from the related tags but I think that we can change it in both to:
```suggestion
      return this.highlightCurated && !!this.suggestion.isCurated;
```